From: FeignInt <8937-Feignint@users.noreply.salsa.debian.org>
Date: Mon, 02 Nov 2020 14:48:50 +0000
Subject: Fix typo in dosbox-staging.metainfo.xml

+ Fixes typo in <description>:
  "variants: IBM PC," => "variants: (IBM PC,"
---
 contrib/linux/dosbox-staging.metainfo.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/linux/dosbox-staging.metainfo.xml b/contrib/linux/dosbox-staging.metainfo.xml
index b4a15ce..03339ad 100644
--- a/contrib/linux/dosbox-staging.metainfo.xml
+++ b/contrib/linux/dosbox-staging.metainfo.xml
@@ -15,7 +15,7 @@
     <p>It features:</p>
     <ul>
       <li>A built-in DOS-like console</li>
-      <li>Emulation of several PC variants: IBM PC, IBM PCjr, Tandy 1000),
+      <li>Emulation of several PC variants: (IBM PC, IBM PCjr, Tandy 1000),
           and CPUs (286, 386, 486, and Pentium I)</li>
       <li>Graphics chipsets: Hercules, CGA, EGA, VGA, and SVGA</li>
       <li>Audio solutions: PC Speaker, Tandy Sound System, Disney Sound Source,
